content/blog/tidyclust-0-1-0/index.Rmd

```diff
@@ -0,0 +1,150 @@
+---
+output: hugodown::hugo_document
+
+slug: tidyclust-0-1-0
+title: tidyclust is on CRAN
+date: 2022-12-06
+author: Emil Hvitfeldt
+description: >
+    Tidyclust is on CRAN. tidyclust provides a common interface for specifying 
+    clustering models, in the same style as parsnip.
+
+photo:
+  url: https://unsplash.com/photos/4Xy08NbMBLM
+  author: Ankush Minda
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, tidyclust]
+---
+
+<!--
+TODO:
+* [X] Look over / edit the post's title in the yaml
+* [X] Edit (or delete) the description; note this appears in the Twitter card
+* [X] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [X] Find photo & update yaml metadata
+* [X] Create `thumbnail-sq.jpg`; height and width should be equal
+* [X] Create `thumbnail-wd.jpg`; width should be >5x height
+* [X] `hugodown::use_tidy_thumbnails()`
+* [X] Add intro sentence, e.g. the standard tagline for the package
+* [X] `usethis::use_tidy_thanks()`
+-->
+
+We're very pleased to announce the release of [tidyclust](https://tidyclust.tidymodels.org/) 0.1.0. tidyclust is the tidymodels extension to fit and use clustering models.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("{package}")
+```
+
+This blog post will introduce tidyclust, how to use it with the rest of tidymodels, and how we can interact and evaluate the fitted cluster models.
+
+```{r setup}
+library(tidymodels)
+library(tidyclust)
+```
+
+## Specifying cluster models
+
+The first thing we need to do is decide on the type of clustering model we want to fit. The pkgdown site provides a [list of all clustering specifications](https://tidyclust.tidymodels.org/reference/index.html#specifications) provided by tidyclust. We are slowly adding more types of models and [suggestions in issues](https://github.com/tidymodels/tidyclust/issues) are highly welcome!
+
+We will use a K-Means model for these examples using `k_means()` to create a specification. tidyclust as with the rest of tidymodels tries to use informative names for functions and arguments, so we will be using `num_clusters` instead of `k`
+
+```{r}
+kmeans_spec <- k_means(num_clusters = 4) %>%
+  set_engine("ClusterR")
+kmeans_spec
+```
+
+We can use the `set_engine()`, `set_mode()`, and `set_args()` we are familiar with from parsnip. The specification itself isn't worth much if we don't apply it to some data. We will use the ames data set from the modeldata package
+
+```{r}
+data("ames", package = "modeldata")
+```
+
+this data set contains a number of categorical variables that unaltered can't be used with a k-means model. Some light preprocessing can be done using the recipes package.
```

Comment:
```suggestion
This data set contains a number of categorical variables that unaltered can't be used with a K-Means model. Some light preprocessing can be done using the recipes package.
```

You capitalize the "K" and "M" earlier on the post—either works, but be consistent. :)